### DISCUSSION TITLE
Reset to default after container restart

### DISCUSSION THREAD

**Discussion by shiftersbox** [polonel/trudesk#572] [Q&A]
Hi @polonel,

Trudesk docker container was inaccessible and when I restarted the container I could access it again but was reset to default as if it was a newly installed container. This happened twice already.

What could I have had missed out during configuration?

<img width="424" alt="Screen Shot 2022-10-07 at 6 08 19 AM" src="https://user-images.githubusercontent.com/114937288/194427785-f0ce77fe-5cde-4ae8-aab2-64a1c17c26ff.png">

**Comment by polonel** [comment]
Did you follow the Docker deployment guide?

https://docs.trudesk.io/v1.2/getting-started/deployment/docker-deployment

**Reply by shiftersbox** [reply]
Used this method with no errors.

<img width="853" alt="Screen Shot 2022-10-09 at 4 38 17 AM" src="https://user-images.githubusercontent.com/114937288/194726977-dcb29ceb-878c-47d1-8bf9-22c3b5f86b88.png">
